src/codemagic/firebase/resources/abstract_resource.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC
+from dataclasses import dataclass
+
+
+@dataclass
+class AbstractResource(ABC):
+    label: str
```

Comment:
I don't follow what the `label` should hold exactly. Isn't it a class variable? If so, please declare it as such.